lib/bugsnag.js

```diff
@@ -101,6 +101,18 @@ module.exports = Bugsnag = (function() {
     }, autoNotifyCallback(err));
   };
 
+  Bugsnag.koaHandler = function(err, ctx) {
+    var request;
+    Configuration.logger.info("Handling koa error: " + (err.stack || err));
+    request = ctx.req;
+    request.protocol = ctx.request.protocol;
```

Comment:
@dtaniwaki Given that we haven't written a better Koa support since yours, I'm going to merge your pull request back in and release a new version.

Thanks!